### DISCUSSION TITLE
Julia extension

### DISCUSSION THREAD

**Discussion by blaylockbk** [coder/code-server#3050] [Q&A] [answered]
Just getting started with Julia, and I see the `Julia (insider)` extension is available in the code-server marketplace, but not the stable `Julia` extension. Is a reason for that? @cmoog

**Comment by jsjoeio** [comment, marked as answer]
cmoog is no longer involved with the project. 

Not sure what the reasoning could be for that, but if you're looking to install the stable `Julia` extension, we suggest using OpenVSX:

`SERVICE_URL=https://open-vsx.org/vscode/gallery ITEM_URL=https://open-vsx.org/vscode/item code-server --install-extension julialang.language-julia`

**Reply by blaylockbk** [reply]
Thanks for the reply and the info. I'll give that a try.
